go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/pressly/goose/v2
```

Comment:
Why do we need `/v2` ?

Does the consumer of this package need to reference "v2" in all import statements too?

Reply:
If there's no `/v2` in the module name, vgo can use either tip of master or latest v1.*.* release. Yes, all the import statements will need this `/v2`, too.

> When developers reach v2, semantic import versioning means that a /v2/ is added to the import path at the end of the module root prefix: my/thing/v2/sub/pkg. There are good reasons for this convention, as described in the earlier post, but it is still a departure from existing tools. Realizing this, vgo will not use any v2 or later tag in a source code repository without first checking that it has a go.mod with a module path declaration ending in that major version (for example, module "my/thing/v2"). Vgo uses that declaration as evidence that the author is using semantic import versioning to name packages within that module. This is especially important for multi-package modules, since the import paths within the module must contain the /v2/ element to avoid referring back to the v1 module. 

-- https://research.swtch.com/vgo-module

Reply:
Goose don't have version tag eat. Please add tag `v0.1.0` or `v1.0.0` and use `module module github.com/pressly/goose`

Reply:
This repo has git tags. Latest is v2.3.0.

Reply:
Oh. Sorry. Yes, vgo want major version in module.